src/InlineObjectCreator/widget/InlineObjectCreator.js

```diff
@@ -114,29 +125,30 @@ mendix.widget.declare('InlineObjectCreator.widget.InlineObjectCreator', {
 			if (this.anim != null)
 				anim.stop();
 			
-			dojo.html.set(this.feedbackNode, msg);
-			dojo.style(this.feedbackNode, {
+			html.set(this.feedbackNode, msg);
+			domStyle.set(this.feedbackNode, {
 				height : 'auto', display : 'block'
 			});
 
-			dojo.animateProperty({
+			fx.animateProperty({
 				node : this.feedbackNode,
 				properties : { height : 0 },
 				duration : 700,
 				delay : 2000,
-				onEnd : dojo.hitch(this, function() {
-					dojo.style(this.feedbackNode, 'display', 'none');
+				onEnd : lang.hitch(this, function() {
+					domStyle.set(this.feedbackNode, 'display', 'none');
 				})
 			}).play();
 		}
 		else
-			dojo.style(this.feedbackNode, 'display', 'none');
+		domStyle.set(this.feedbackNode, 'display', 'none');
 	},
 	
 	keypress : function(e) {
 		if (e.keyCode == 13 && (this.rows == 1 || e.ctrlKey == true)) { //multiline needs ctrl key
 			this.trigger();
-			dojo.stopEvent(e); //no bubbling
+			e.preventDefault(); // no bubbling
+			e.stopPropagation(); // no bubbling

```

Comment:
require `dojo/_base/event` and use `event.stop(e);`

Reply:
Done